php/2020-SDK/Voice/Gather-Voice/composer.lock

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
It might be cleaner to not include this in the sample app, but probably doesn't matter too much

Reply:
Would including the lock file prevent sample apps from breaking when sdk's are changed if we fail to maintain them?